authorization_services/topics/enforcer-claim-information-point.adoc

```diff
@@ -0,0 +1,165 @@
+[[_enforcer_claim_information_point]]
+= Claim Information Point
+
+A Claim Information Point (CIP) is responsible for resolving claims and pushing these claims to the {project_name} server in order
+to provide to policies more information about the access context. They can be defined as a configuration option to the `policy-enforcer`
+in order to resolve claims from different sources, such as:
+
```

Comment:
A Claim Information Point (CIP) is responsible for resolving claims and pushing these claims to the {project_name} server in order to provide more information about the access context to policies. They can be defined as a configuration option to the `policy-enforcer` in order to resolve claims from different sources, such as: